connectors/byoei.py

```diff
@@ -471,3 +486,9 @@ async def async_bulk(
             "doc_deleted": fetcher.total_docs_deleted,
             "fetch_error": fetcher.fetch_error,
         }
+
+    def _get_expand_wildcards(index_name):
```

Comment:
if this is part of the class, you want:

```
def _get_expand_wildcards(self, index_name):
```

Reply:
I also wonder why you used that name for this function. shouldn't this be called `index_type()` ?

Reply:
Yeah, I think this name makes much more sense! It was just simple pull-to-a-method refactoring and I did not think much of the method name